### ISSUE TITLE
typo markdown/podman-pull.1.md

### ISSUE THREAD

**Issue by dilyanpalauzov** [containers/podman#11536]
https://github.com/containers/podman/blob/main/docs/source/markdown/podman-pull.1.md says:
```
OPTIONS
--all-tags, a

All tagged images in the repository will be pulled.
```
`a` should be prefixed by a hyphen.

**Comment by rhatdan** [comment]
Actually the `, a` should be removed --all-tags and -a are NOT aliases for the same option.

**Comment by giuseppe** [comment]
closed with #11537
